src/chat/chatter.py

```diff
@@ -308,24 +308,24 @@ def updateGame(self):
             elif game.state == GameState.PLAYING:
                 self.statusItem.setIcon(util.THEME.icon("chat/status/playing.png"))
                 self.statusItem.setToolTip("Playing Game<br/>"+url.toString())
-
-            # We're in game, show the map if toggled on
-            if util.settings.value("chat/chatmaps", False):
-                mapname = game.mapname
-                icon = maps.preview(mapname)
-                if not icon:
-                    self.chat_widget.client.downloader.downloadMapPreview(mapname, self.mapItem)  # Calls setIcon
-                else:
-                    self.mapItem.setIcon(icon)
-
-                self.mapItem.setToolTip(game.mapdisplayname)
-            else:
-                self.mapItem.setIcon(QtGui.QIcon())
-                self.mapItem.setToolTip("")
         else:
             self.statusItem.setIcon(QtGui.QIcon())
             self.statusItem.setToolTip("Idle")
 
+        # Map icon handling - if we're in game, show the map if toggled on
```

Comment:
I like this. If we already pulled it out, maybe we can put it in a separate method altogether, like update_game_map_icon(self, game).

Reply:
That might make sense if we had a extra route for map change, else it's closely related to game.state. The signaling for seperate mehtods would be ... complicated.

(while we're here: Does self._verifySortOrder() makes any sense here in updateGame? Nothing related to sorting with state & map icon.

Reply:
We don't need to make a separate control flow for the method, we can just call it from here. It'd be just a code organization matter, to keep functions small and focused on one thing.
_verifySortOrder is just a dirty fix to ensure that chatters are well-sorted after some updates (downside of using qt widget lists instead of qt models like games are done now). Map updates don't affect the order, so we don't need to call it here.